### DISCUSSION TITLE
Use Case overview

### DISCUSSION THREAD

**Discussion by wenjing** [trustoverip/tswg-authentic-ai#16] [General]
The original Google doc is here:
https://docs.google.com/document/d/1Snga-dkUG0XaLaXx1q9FKojaFy42CTZhvyFJbo9KgLU/edit
We have moved all use cases one by one to GitHub discussion.

This topic covers the preceding overview text:

Authentic Generative AI Proposal (Updated) - to review on Feb 15, 2024.
(to be expanded into a short white paper for intro and circulation)

Latest news shows an urgent need for methods that provide authenticity information for the creation and distribution of AI generated, AI & human, or human generated content.
The AI and Metaverse Task Force is launching a new initiative to produce a specification for such a method utilizing TSP and C2PA, where C2PA provides provenance information (for some content types) and TSP supports authentic (& optionally confidential, private) distribution of such content by anybody. This is illustrated in the diagram below.
Why and what’s new: Compared to the typical web service model, TSP is better suited for any-to-any communications (in other words, decentralized and distributed across boundaries); more scalable; designed to support bots/agents.
Proposed work scope: 
use cases /  application scenarios (labeling, transparency, auditing, authorship, data collection consent, copyright, …)
method specification, 
trust property considerations.
What is not in scope: 
Proposed schedule: 6 months after launch.

Use Cases: (Please contribute your use case descriptions below, append at the end of the list. Please try to provide concrete information e.g. on (i) what is the problem, (ii) why the existing methods aren’t working, (iii) why and how the proposed AGAI work may help. Bullet points are fine.

<img width="710" alt="Screenshot 2024-04-24 at 5 23 52 PM" src="https://github.com/trustoverip/authentic-ai/assets/430107/551c6fb1-94be-4476-aab8-b5cf58be68d1">

**Comment by guru-barbes** [comment]
Based on Wenjing's initial Use Case structure, I've added a few more sections to consider for a template:

**1. Context**
Please try to provide concrete information e.g. on
(i) what is the problem: 
(ii) why the existing methods aren’t working: 
(iii) why and how the proposed AGAI work may help: 

**2. Actors**
Who are the main actors in this use case? And how might they use TSP+C2PA to accomplish some goal, solve some problem, provide some benefit?

**3. Data**
What kind of data is at the heart of this use case? What kind of data is being captured (from where), exchanged, generated, processed …?

**4. Governance**
What are the governance considerations and implications of this use case?

**5. AI Models**
What are the specific AI models that are being applied to this use case, or could be applied to this use case?

**6. Output**
Where / how does the content output get applied, consumed or otherwise used?

**Reply by neiljthomson** [reply]
Data - for exchange and human validation, separate processing, requirements on data being ingested, exchanged and output from AI systems (w humans, non-AI or diff types of AI)
